include/zephyr/drivers/dma/dma_mcux_pxp.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_DMA_MCUX_PXP_H_
+#define ZEPHYR_INCLUDE_DRIVERS_DMA_MCUX_PXP_H_
+
+#define DMA_MCUX_PXP_CMD_MASK (0xE0)
+#define DMA_MCUX_PXP_CMD_SHIFT (0x5)
+
+#define DMA_MCUX_PXP_FMT_MASK (0x1F)
+#define DMA_MCUX_PXP_FMT_SHIFT (0x0)
```

Comment:
Please do not add unnecessary parentheses.

Reply:
Noted, fixed in latest push